devtools/common-core/src/main/java/io/quarkus/dependencies/Extension.java

```diff
@@ -175,6 +177,12 @@ public String gav() {
         return managementKey() + ":" + version;
     }
 
+    private static final Pattern QUARKUS_PREFIX = Pattern.compile("^quarkus-");
+
+    public String getSimplifiedArtifactId() {
```

Comment:
It might be overkill,  but would it be worth storing the "simplified" artifactId so the matcher isn't used on every call?

Maybe setting it as part of construction or on `setArtifactId`?